develop/devguide/Automation/UIBlockTypesOverview.md

```diff
@@ -270,6 +270,13 @@ All files uploaded by users will by default be placed in the *C:\\Skyline DataMi
 > [!TIP]
 > See also: [GetUploadedFilePath](xref:Skyline.DataMiner.Automation.UIResults#Skyline_DataMiner_Automation_UIResults_GetUploadedFilePath_System_String_)
 
+> [!NOTE]
+> Contrary to the other UI block types, the FileSelector does not allow to set an initial value. During an interactive automation script session, it is however possible (since RN37620) to keep the files that where already uploaded after the UI was shown.
```

Comment:
"initial value" can maybe also be turned into a url that links to the page with more info (like we did for DestVar)

Reply:
Checked with @MariekeGO and she agreed. They've been added in 38a9a95